cg/meta/workflow/taxprofiler.py

```diff
@@ -110,18 +123,65 @@ def write_sample_sheet(
                 ),
             )
 
+    def write_params_file(self, case_id: str, dry_run: bool = False) -> None:
+        """Write params-file for taxprofiler analysis in case folder."""
+        default_options: Dict[str, str] = self.get_default_parameters(case_id=case_id)
+        if dry_run:
+            return
```

Comment:
Do you mean to remove these two lines from `write_params_file` and add the following to `config_case`?

    
  ```
       if not dry_run:
            LOG.info("Generating parameters file")
            self.write_params_file(case_id=case_id, dry_run=dry_run)
        if dry_run:
            LOG.info("Dry run: Config files will not be written")
            return

```

Reply:
You can simply move it after the `dry_run` in `config_case`, since you have a return inside the if statement:

```
        if dry_run:
            LOG.info("Dry run: Config files will not be written")
            return
        self.write_params_file(case_id=case_id, dry_run=dry_run)
```